programs/nodeos/main.cpp

```diff
@@ -119,7 +120,7 @@ int main(int argc, char** argv)
          .default_unix_socket_path = "",
          .default_http_port = 8888
       });
-      if(!app().initialize<chain_plugin, net_plugin, producer_plugin, resource_monitor_plugin>(argc, argv)) {
+      if(!app().initialize<blockvault_client_plugin, chain_plugin, net_plugin, producer_plugin, resource_monitor_plugin>(argc, argv)) {
```

Comment:
Not sure you actually need this since chain_plugin already REQUIRES() blockvault_client_plugin

Reply:
I don't think chain_plugin should require blockvault_client_plugin unless I'm missing something.